grails-app/services/org/pih/warehouse/data/LoadDataService.groovy

```diff
@@ -69,7 +70,7 @@ class LoadDataService {
 
             Organization organization = new Organization(
                     name: organizationName,
-                    code: partyRole.substring(0, 3), // FIXME: Should code be provided or generated?
+                    code: identifierService.generateOrganizationIdentifier(organizationName),
```

Comment:
Whoa, do we know what PartyRole.substring(0, 3) was doing? Seems like it might return the uuid id of the party role? I think your change is fine, but given that it's not part of the requirements we should make a note of this in the ticket / discuss with Manon, Kelsey, et al. 

Given that this is demo data, I don't know if anyone is going to care. It's just a good idea to discuss so we can test this properly.